scene/resources/material.cpp

```diff
@@ -2836,6 +2836,64 @@ Ref<Material> BaseMaterial3D::get_material_for_2d(bool p_shaded, Transparency p_
 	return materials_for_2d[key];
 }
 
+Dictionary BaseMaterial3D::get_material_from_texture_path(const String &p_file_path) const {
```

Comment:
This seems like it should be something in the editor plugin. Is there a reason why this has to be in the material class itself?

Reply:
Not really, it's a leftover from earlier development so I should be able to move it to the editor plugin.